index.html

```diff
@@ -30,16 +43,17 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <!-- added an id to select each box individually -->
+                <div id="box1" class="box">Box 1</div>
+                <div id="box2" class="box">Box 2</div>
+                <div id="box3" class="box">Box 3</div>
+                <div id="box4" class="box">Box 4</div>
+                <div id="box5" class="box">Box 5</div>
+                <div id="box6" class="box">Box 6</div>
+                <div id="box7" class="box">Box 7</div>
+                <div id="box8" class="box">Box 8</div>
+                <div id="box9" class="box">Box 9</div>
+                <div id="box10" class="box">Box 10</div>
```

Comment:
Great job, another alternative would be to use the nth-child selector so you don't have to modify the classes or ids